src/ashwini/A20_SumOfChar.java

```diff
@@ -0,0 +1,27 @@
+package ashwini;
+
+public class A20_SumOfChar {
+
+	int digitCount = 0;
+	int sum = 0;
```

Comment:
if this instance variable is useful at multiple place then only we should write at common place, just FYI